code/datums/flock/flock.dm

```diff
@@ -560,6 +570,21 @@ var/flock_signal_unleashed = FALSE
 	var/enemy_name = M
 	return (enemy_name in src.enemies)
 
+/datum/flock/proc/addAlly(atom/A)
+	if (ismob(A) && find_radio_on(A))
+		boutput(A, "<i class='flocksay'>You hear echoes of the signal in your mind. In some way you can't quite explain, they seem to approve of you.</i>")
```

Comment:
For \<i class='flocksay'\>, is this just italic or? Relay code uses \<span class='flocksay italic'\>

Reply:
yep it's italic, not sure which would be preferred.

Reply:
Would maybe go with second just to be consistent but not necessary